Code/Conviva/Source/Events/ConvivaReporters/BasicEventConvivaReporter.swift

```diff
@@ -138,3 +136,10 @@ class BasicEventConvivaReporter: BasicEventProcessor {
         reportEndedIfPlayed()
     }
 }
+
+extension CISVideoAnalytics {
+    func safeSetContentInfo(_ contentInfo: [AnyHashable : Any]) {
+        let filteredContentInfo: [AnyHashable : Any] = contentInfo.filter { self.getMetadataInfo()[$0.key] == nil }
```

Comment:
What is the main goal of this PR? Prevent a metadata field from being set when it already has a value in place?
Does this break some logic somehow?